federation/backends/api.md

```diff
@@ -4,6 +4,76 @@ The general contract is the [openEO API](https://api.openeo.org) in the latest s
 
 The aggregator that proxies the back-ends in the federation also implements the same API, but it also implements the "Federation Extension" (currently in draft state).
 
+## Profiles
+
+In addition to the general openEO API specification and their [API Profiles](https://openeo.org/documentation/1.0/developers/profiles/api.html),
+openEO Platform requires to implement an additional API profile:
+*LP: Required for openEO Platform*, which requires the openEO profile *L2: Recommended*.
+The requirement to implement two of *L1A*, *L1B*, and *L1C* has been restricted for openEO Platform to always require *L1A: Synchronous Processing* and *L1B: Batch Jobs* (see req. no. 703). This means that *L1C: Secondary Web Services* is optional.
+
+<figure>
+    <img src="./profiles/api.png" alt="The hierarchy of openEO and openEO Platform API profiles.">
+    <figcaption>An overview of the openEO and openEO Platform API profiles.</figcaption>
+</figure>
+
+### LP: Required for openEO Platform
+
+The profile only lists requirements that are not covered by the openEO profile *L2: Recommended* yet.
+
+#### API fundamentals
+
+| # | Functionality | Description |
+| -- | -- | -- |
+| 14 | All > Billing | Supports the openEO Platform credit system |
+
+#### File Formats
+
+| # | Functionality | Description |
+| -- | -- | -- |
+| 45 | `GET /file_formats` | File format names and parameters aligned with openEO Platform (at least GTIFF and NetCDF) |
```

Comment:
```suggestion
| 45 | `GET /file_formats` | File format names and parameters aligned with openEO Platform (at least for GTIFF and NetCDF if supported) |
```

Does that mean you need to support GTIFF and NetCDF or if you support any of both your names and parameters need to be aligned. I suspect latter but find it no 100% clear.

Reply:
It was meant to only require the alignment of naming and parameters as defined in https://docs.openeo.cloud/federation/backends/fileformats.html#best-practices-file-formats

```suggestion
| 45 | `GET /file_formats` | File format names and parameters aligned with openEO Platform as defined for the pre-defined [file formats](./fileformats.md) |
```